ops/ansible/roles/bfd-server/defaults/main.yml

```diff
@@ -30,3 +30,6 @@ data_server_v2_enabled: false
 
 # Whether or not partially adjudicated claims data resources are enabled.
 pac_resources_enabled: false
+
+# Configures the type of enabled source types for PAC claims
+pac_claim_source_types: 'fiss,mcs'
```

Comment:
Is this configuration still used, I thought all of our server params are being loaded from ops/terraform/services/base?

Reply:
It's a good question, though one I think for the BFD team to weigh in on

Reply:
@mjburling should we still be making changes to the ansible yaml files or are these deprecated at this point and configuration value changes only need to be made to the terraform value yaml files?

Reply:
@ericbuckley This is still a valid location for ansible variables that must be defined **and** have a reasonable default. While these values aren't leveraged _generally_ in our path-to-production environments of `test`, `prod-sbx`, and `prod`, the ansible test harness (`test_basic.yml`) continues to defer to them unless they're explicitly defined when calling the role via `import_role`–if they're required by ansible but are not defined by either a default value or explicitly when calling the role, ansible will fail.